text/0018-genesis_block.md

```diff
@@ -0,0 +1,94 @@
+- Proposal Code Name: genesis_block
+- Start Date: 2019-09-27
+- NEP PR: [nearprotocol/neps#0000](https://github.com/nearprotocol/neps/pull/0000)
+
+# Summary
+[summary]: #summary
+
+Genesis block will contain:
+* Accounts and smart contracts required to describe the initial ownership, vesting schedule, and the lockup of the tokens;
+* Economics config;
+* Accounts from the testnet.
+
+# Token ownership
+We want Near investors and token holders to have their vesting schedule, lock up period and ability to stake to be implemented through the accounts and smart contracts on the blockchain.
+
+Each token holder will have tokens that go through the following lifecycle:
+* Unvested;
+* Vested, but locked;
+* Unlocked.
+
+Vesting is done over 4 years with 1 year cliff starting with:
+* For employees -- employment date or blockchain initiation date, whichever is latests;
+* For investors -- investment date.
+
+Non-standard vesting and lock up schedules can be later created by the Near Inc foundation.
+
+Lock up is applied to the token holder with 1 year starting with the mainnet launch. If token is vested before the lock up expires then it cannot be sold until the end of the lock up. If token is vested after the lock up then it can be sold immediately.
+
+In case of employees the NEAR Inc company can decide to terminate the employment and the person would lose the unvested tokens, however the lock period will still be applied to vested tokens.
+
+# Design details
+We create the following accounts and contracts:
+* For each token holder we create an account and a contract;
+* We create one account for NEAR Inc.
```

Comment:
I don't think NEAR Inc will own tokens. The tokens are distributed by the foundation.

Reply:
Indeed: https://docs.nearprotocol.com/docs/overview/what-is-near#the-near-foundation

Reply:
Fixed.